addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12222,7 +12225,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#23050"
-msgid "Activate teletext"
+msgid "Activate Teletext"
```

Comment:
> Haven't changed it back now, as two different people disagreeing on how it should be :)
> 
> If you guys can Agree on Teletext <-> teletext and Zeroconf <-> zeroconf I'm OK with going along here.

So @opdenkamp and @MartijnKaijser for Teletext <-> teletext and @da-anda for Zeroconf <-> zeroconf

Whatever you guys decide on here Im ok.